website/docs/docs/dbt-cloud-apis/sl-jdbc.md

```diff
@@ -147,9 +147,44 @@ select * from {{
 }}
 ```
 
-</DetailsToggle>
+</Expandable>
+
+<Expandable alt_header="Fetch metrics by substring search">
+
+You can filter your metrics to include only those that contain a specific substring (sequence of characters contained within a larger string (text)). Use the `search` argument to specify the substring you want to match.
+
+```sql
+select * from {{ semantic_layer.metrics(search='order') }}
+```
+
+If no substring is provided, the query returns all metrics.
 
-<DetailsToggle alt_header="List saved queries">
+</Expandable> 
+
+<Expandable alt_header="Paginate metadata calls">
+
+In cases where user manifests are large, pagination is a useful way to prevent query character limits in the data platform. You can paginate results for both `semantic_layer.metrics()` and `semantic_layer.dimensions()` calls using the `page_size` and `page_number` parameters.
```

Comment:
"In cases where user manifests are large, pagination is a useful way to prevent query character limits in the data platform. You can paginate results" 

Change to: "In the case when you don't want to return the full result set from a metadata call you can paginate the resluts.."